crypto/src/lib.rs

```diff
@@ -39,7 +39,7 @@ pub const BLS_NORMAL: &str = "bls_normal";
 pub const BLS_SMALL: &str = "bls_small";
 
 /// Algorithm for hashing
-#[derive(Clone, Copy, Debug)]
+#[derive(Debug, Clone, Copy)]
```

Comment:
Anything wrong with the alphabetical order? I'm just curious.

Reply:
Perhaps we could add a minor point to the style guide if having `Debug` before `Clone` and `Copy` has an even minor impact.

Reply:
ahm, not that it really matters, I just tried enforcing some consistency in derive order, my rules were:
1. most frequently used items first
2. if derive depends on another derive that one goes first, like `Clone, Copy` or `PartialEq, Eq`